Tests/Clappr_Tests/Classes/Plugin/Playback/AVFoundationPlaybackTests.swift

```diff
@@ -1056,35 +1072,43 @@ class AVFoundationPlaybackTests: QuickSpec {
                     }
                 }
 
-                it("triggers willSeek event") {
+                it("triggers willSeek event and send position") {
                     let playback = AVFoundationPlaybackMock(options: [:])
                     playback.videoDuration = 100
                     let player = AVPlayerStub()
                     playback.player = player
                     player.setStatus(to: .readyToPlay)
                     var didTriggerWillSeek = false
-                    playback.on(Event.willSeek.rawValue) { _ in
+                    var position: Float64?
+
```

Comment:
I think we can change for Double, to avoid architecture dependency (32 / 64 bits). What do you think?

Reply:
Yep! You're right!
Done